app/src/main/java/org/jtaccuino/app/jshell/extensions/fx/ChartsFx.java

```diff
@@ -51,7 +53,29 @@ public static ScatterChart<?,?> scatterFx(int[] x, double[] y, String xLabel, St
         return scatterChart;
     }
 
-    public static LineChart<?,?> lineFx(int[] x, double[] y, String xLabel, String yLabel, String title) {
+    public static ScatterChart<?, ?> scatterPlot(List<double[]> xData, List<double[]> yData, String xLabel, String yLabel, String[] title) {
+        var xAxis = new NumberAxis();
+        xAxis.setLabel(xLabel);
+
+        var yAxis = new NumberAxis();
+        yAxis.setLabel(yLabel);
+
+        var scatterChart = new ScatterChart<Number, Number>(xAxis, yAxis);
+
+        for (int i = 0; i < xData.size(); i++) {
+            var x = xData.get(i);
+            var y = yData.get(i);
+            var intStream = IntStream.range(0, x.length);
```

Comment:
you might want to ensure that xData and yData have the same length.

Reply:
Will be done with a more general rework to separate out the extension.